docs/reference/run.md

```diff
@@ -298,13 +298,15 @@ non-zero exit status more than 10 times in a row Docker will abort trying to
 restart the container. Providing a maximum restart limit is only valid for the
 **on-failure** policy.
 
-## Exit Status
+## Exit status
 
 The exit code from `docker run` gives information about why the container
-failed to run or why it exited.  When `docker run` exits with a non-zero code,
-the exit codes follow the `chroot` standard, see below:
+failed to run or why it exited. When `docker run` exits with a non-zero code,
+the exit codes follow the `chroot` standard.
```

Comment:
I think the "see below" added _some_ value (it links the standard with the description below).

Perhaps, ", which is described below" or "as follows", or .. LOL something


(either that, or we should look if there's a link with relevant documentation about this "chroot standard")

Reply:
Not sure even calling out a `chroot` standard is very useful. I will simplify.